### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
readme for tac processes folder to hopefully make finding things easier

### CODE REVIEW

File: process/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# TAC Process Documentation #
+
+This folder contains the policies and procedures the OpenSSF TAC uses to perform their duties as well as guidelines for the Technical Initiatives of the Foundation.
+
+## TAC Election Process ##
```
Comment (marcelamelara) [inline process/README.md:5]: I think we should move this section down in the doc. From a TI's perspective, the processes I'd want to find most quickly are the Decision and Lifecycle sections.

File: process/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## TAC Decision Process ##
+[TAC Decision Process](https://github.com/ossf/tac/blob/main/process/TAC-Decision-Process.md)
+
+## Technical Initiative Processes & Lifecycle ##
+- [Technical Initiative Lifecycles](https://github.com/ossf/tac/blob/main/process/Technical_Initiative_Lifecycle.md)
+- [Technical Initiative Temaplates](https://github.com/ossf/tac/tree/main/process/templates)
+- [Technical Initiative "Gives & Gets"](https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md)
+- [Technical Initiative Funding Process](https://github.com/ossf/tac/blob/main/process/TI%20Funding%20Request%20Process.md)
+- [Building an Open Source Community guidelines](https://github.com/ossf/tac/blob/main/process/building-an-open-source-community.md)
```
Comment (marcelamelara) [inline process/README.md:18]: Suggesting a minor re-arrangement of the listed docs to make this a slightly more TI priority oriented list (imo).
```suggestion
- [Technical Initiative Lifecycles](https://github.com/ossf/tac/blob/main/process/Technical_Initiative_Lifecycle.md)
- [Technical Initiative "Gives & Gets"](https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md)
- [Technical Initiative Funding Process](https://github.com/ossf/tac/blob/main/process/TI%20Funding%20Request%20Process.md)
- [Building an Open Source Community guidelines](https://github.com/ossf/tac/blob/main/process/building-an-open-source-community.md)
- [Technical Initiative Lifecycle Templates](https://github.com/ossf/tac/tree/main/process/templates)
```

**Review by marcelamelara — COMMENTED** [review]
Thanks @SecurityCRob, I really like where we're going with this. I have a couple comments on re-arranging some of the sections or bullets to be a bit more oriented towards TI readers.

**Review by lehors — APPROVED** [review]
I like this! Thank you.

**Review by marcelamelara — APPROVED** [review]
LGTM, thanks!

**Pull request merged**